.github/workflows/python-package.yml

```diff
@@ -56,20 +40,24 @@ jobs:
         run: poetry run ruff . --line-length 132 --ignore E501,E402
 
       - name: Run unit tests
+        # Nektos act runs tests as root;
+        # and failing from line 10 on web/web/settings.py
+        env:
+          CAPE_AS_ROOT: 1
```

Comment:
As mentioned in the comment, when running the github action locally, using nektos/act, the job runs as the root user.

settings.py checks to ensure we're not the root user, unless the `CAPE_AS_ROOT` environment variable is set; if not set, the tests are not running. I wanted to avoid changing settings.py.